_opensearch/query-dsl/compound/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Compound queries
+parent: Query DSL
+has_children: true
+nav_order: 40
+---
+
+# Compound queries
```

Comment:
Curious about the order of the query types in the TOC- should it be alphabetical? (The order in the overview topic is in alphabetical.)

Reply:
Ordered loosely by popularity.